.gitignore

```diff
@@ -51,6 +51,9 @@ qtcreator-*
 # Vim
 *.swp
 
+# JetBrains IDEs (e.g. Pycharm or CLion)
+.idea/
+
```

Comment:
@rhaschke argued against more entries here before, but I'm ok with adding jetbrains to the list when vscode and qtcreator are there already.

Reply:
I'm pointing people to a better approach. But I don't mind adding JetBrains here as well.